tests/integration/docidcache/test_crud_cache.py

```diff
@@ -0,0 +1,314 @@
+import os
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+from jina import Flow, Document
+from jina.executors.indexers import BaseIndexer
+from jina.executors.indexers.cache import DocIDCache
+from jina.executors.indexers.keyvalue import BinaryPbIndexer
+from jina.executors.indexers.vector import NumpyIndexer
+
+cur_dir = Path(os.path.dirname(os.path.abspath(__file__)))
+
+KV_IDX_FILENAME = 'kv_idx.bin'
+VEC_IDX_FILENAME = 'vec_idx.bin'
+DOCS_TO_SEARCH = 1
+TOP_K = 5
+BATCH_SIZE = 4
+DOCS_TO_INDEX = 10
+
+pytest.register_assert_rewrite('check_indexers_size')
```

Comment:
what does this do? just curious?

Reply:
I was trying to fix a problem with pytest not showing me the full assert location in PyCharm. Thought this would help. Didn't. Will remove